### PR TITLE
Bug 1218913 - added missing type entries to schemas

### PR DESCRIPTION
**Pull request by petemoore**
Thanks!

### CODE REVIEW

File: schemas/artifact-created-message.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ properties:
     description: |
       Identifier for the worker within which the run with the created artifact
       is running.
-    type: "string"
+    type:         string
```
Comment (petemoore) [inline schemas/artifact-created-message.yml]: This is just to keep consistent formatting

File: schemas/artifact-created-message.yml [hunk 2]
```diff
@@ -32,12 +32,14 @@ properties:
     title:        "Artifact Created"
     description: |
       Information about the artifact that was created
+    type:         array
```
Comment (jonasfj) [inline schemas/artifact-created-message.yml]: type object if I'm not mistaken it has properties..

File: schemas/constants.js [hunk 3]
```diff
@@ -79,6 +79,7 @@ module.exports = {
   // Message version numbers
   "message-version": {
     "description":  "Message version",
+    "type":         "string",
```
Comment (jonasfj) [inline schemas/constants.js]: integer

File: schemas/artifact-created-message.yml (hunk 1 above)
Comment (jonasfj) [inline schemas/artifact-created-message.yml]: nice! :)

File: schemas/artifact-created-message.yml (hunk 2 above)
Comment (petemoore) [inline schemas/artifact-created-message.yml]: whoooooops

File: schemas/constants.js (hunk 3 above)
Comment (petemoore) [inline schemas/constants.js]: whoooooops again

**Pull request merged**